### PR TITLE
 adding user guide for telemetry

### PR DESCRIPTION
**Pull request by NajmudheenCT**
adding user guide for telemetry , which is step to step guide to install, configure , view telemetry data

### CODE REVIEW

**Comment by NajmudheenCT** [comment]
@skdwriting @xing-yang Please review

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+By following this guide you will be able to install opensds with telemetry tools, create test volumes, create sample grafana dashboard, create sample alert rules and view dashboards.
+
+## Deploying Opensds with Telemetry
+Follow this opensds wiki link to install opensds with telemetry and 
+***Pay special attention to the section below before starting the deploy operation!***
+
+**Download opensds-installer code**
+
+    ```
+
```
Comment (xxwjj) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: remove empty line

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+     git clone https://github.com/opensds/opensds-installer.git
+    cd opensds-installer/ansible
+    # Checkout the latest release, give master by defualt or specfy the branch 
+    git checkout master
+    ```
+    
+  **How to enable Telemetry installation**
+  
+   Update the file : ansible/group_vars/telemetry.yml for install_telemetry_tools
+
```
Comment (xxwjj) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: same here

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+    
+  **How to enable Telemetry installation**
+  
+   Update the file : ansible/group_vars/telemetry.yml for install_telemetry_tools
+   
+   
+    
+    ```
+    # Do you need to install telemetry tools?
+    install_telemetry_tools: true
```
Comment (xxwjj) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: s/install_telemetry_tools/enable_telemetry_tools

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+## Deploying Opensds with Telemetry
+Follow this opensds wiki link to install opensds with telemetry and 
+***Pay special attention to the section below before starting the deploy operation!***
+
+**Download opensds-installer code**
+
+    ```
+    
+     git clone https://github.com/opensds/opensds-installer.git
+    cd opensds-installer/ansible
```
Comment (xxwjj) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: change to ```cd opensds-installer``` is better.

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 1 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: fixed! Thanks

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 2 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: Fixed

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 3 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: modified text, config present is /ansible/groupvars/telemetry.yml

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 4 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: fixed, Thanks

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 3 above)
Comment (leonwanghui) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: It's still not changed.

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+    
+Detailed instruction is in this link
+
+- [OpenSDS Local Cluster Installation through Ansible On Ubuntu](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) (Recommended)
+
+### Make environment ready for metric collection
+
+**export required OPENSDS environment variables**
+```
+export OPENSDS_ENDPOINT=http://{{ apiserver_cluster_ip }}:50040
```
Comment (leonwanghui) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: Here is not apiserver_cluster_ip, instead it's host_ip

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+export OPENSDS_AUTH_STRATEGY=keystone
+export OS_AUTH_URL=http://{{ authchecker_cluster_ip }}/identity
```
Comment (leonwanghui) [inline readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: Same here.

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 3 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: This is the text as in install script.We just mention it here ofr user awareness.   Are you asking to change the field name in install scripts?

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 5 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: Changed, Thanks

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 6 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: Changed

File: readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md (hunk 3 above)
Comment (NajmudheenCT) [reply readthedocs/starterguides/userguides/Telemetry_with_LVM_UserGuide.md]: @leonwanghui Please see my comment above. Let me know if you have any quesion.

**Pull request merged**